compose/types/module_field_options.go

```diff
@@ -13,8 +13,13 @@ type (
 )
 
 const (
+	moduleFieldOptionExpression         = "expression"
 	moduleFieldOptionIsUnique           = "isUnique"
 	moduleFieldOptionIsUniqueMultiValue = "isUniqueMultiValue"
+
+	moduleFieldNumberOptionPrecision         = "precision"
+	moduleFieldNumberOptionPrecisionMin uint = 0
+	moduleFieldNumberOptionPrecisionMax uint = 6
```

Comment:
Should we define this globally? I don't see why we would want to have different precision limits over different parts of the system